src/test/java/no/unit/nva/database/DatabaseAccessor.java

```diff
@@ -25,10 +29,14 @@
 public abstract class DatabaseAccessor implements WithEnvironment {
 
     public static final String USERS_AND_ROLES_TABLE = "UsersAndRolesTable";
-    public static final String HASH_KEY_NAME = "PK1A";
-    public static final String SORT_KEY_NAME = "PK1B";
-    private static final Long CAPACITY_DOES_NOT_MATTER = 1000L;
+    public static final String HASH_KEY_NAME = "PrimaryKeyHashKey";
```

Comment:
Hvorfor bruker du ikke DatabaseIndexDetails.java? Er det for isolasjon?

Reply:
fixed :)